faq/billing.mdx

```diff
@@ -69,6 +69,16 @@ Invoices are issued each month based on the resources' consumption of the previo
 Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone.
 The automatic payment of invoices occurs in the first few days of a month, once the invoice is issued and is processed by using the preferred payment method of your account.
 
+## What does the status of my invoice mean?
+
+The status of an invoice can be:
+* **Draft**: Concerns the invoice for the current calendar month. The billing period has not yet finished, so the invoice is not finalized and is subject to change.
```

Comment:
```suggestion
* **Draft**: regards the invoice for the current calendar month. The billing period has not yet finished, so the invoice is not finalized and is subject to change.
```

Reply:
I'm not sure about this suggestion, I think concerns makes more sense than regards?

Reply:
+1 with @RoRoJ